README.md

```diff
@@ -220,6 +223,16 @@ apiaudio.Script.create()
 
 Same logic applies for other resources (`Speech`, `Voice`, `Sound`...)
 
+### `Organization` resource <a name = "organization"> </a>
```

Comment:
Looks good but can you format the same as the other methods? sorry to be pedantic :D

Reply:
I mean format the documentation the same